horovod/mxnet/mpi_ops.cc

```diff
@@ -202,68 +202,58 @@ extern "C" int horovod_mxnet_allreduce_async(
       RunContext rctx, Engine::CallbackOnComplete cb) mutable {
       DoAllreduce(input, output, average, new_name, cb);
   };
+#if HAVE_CUDA
```

Comment:
The way that github shows the difference is hard to follow. Let's discuss offline about these changes tomorrow.

Reply:
Discussed offline.